crates/redis_interface/src/lib.rs

```diff
@@ -155,13 +155,6 @@ impl RedisConnectionPool {
     }
 }
 
-impl Drop for RedisConnectionPool {
```

Comment:
Why this change is reverted?

Reply:
There was a breaking change, Reverted to test, No Re-reverted after hotfix in #1063